docs/opcodes/5E.mdx

```diff
@@ -0,0 +1,39 @@
+---
+fork: Cancun
+group: Stack Memory Storage and Flow Operations
+---
+
+*Index 1 is top of the stack.*
+
+## Notes
+
+Copying takes place as if an intermediate buffer was used, allowing the destination and source to overlap.
+If `length > 0` and `(src + length or dst + length)` is beyond the current memory length, the memory is extended with respective gas cost applied.
```

Comment:
We are mixing `length` and `size` here 🤔  (using `length` here, but `size` elsewhere)

I think we should stick to one of them